spec/system/distribution_system_spec.rb

```diff
@@ -150,6 +261,17 @@
 
         expect do
           click_button "Save", match: :first
+
+          expect(page).to have_selector('#distributionConfirmationModal')
+          within "#distributionConfirmationModal" do
+            expect(page).to have_content("You are about to create a distribution for")
+            expect(find(:element, "data-testid": "distribution-confirmation-partner")).to have_text(partner.name)
+            expect(find(:element, "data-testid": "distribution-confirmation-storage")).to have_text(storage_location.name)
```

Comment:
Can we hardcode the names? Faker has bitten us with this before. :)

Reply:
In the case of `spec/system/distribution_system_spec.rb`, the storage location and partner names are defined with `let` and FactoryBot near the top of the file:

```ruby
let(:storage_location) { create(:storage_location, organization: organization) }
let!(:partner) { create(:partner, organization: organization) }
```

Then used throughout, for example:

```ruby
select partner.name, from: "Partner"
select storage_location.name, from: "From storage location"
```

Did you mean to change the FactoryBot create to pass in specific names and then replace throughout the entire spec file? Something like:

```ruby
let(:storage_location) { create(:storage_location, organization: organization, name: "Test Storage Location") }
let!(:partner) { create(:partner, organization: organization, name: "Test Partner") }
...

# Later in tests

select "Test Partner", from: "Partner"
select "Test Storage Location", from: "From storage location"
...
expect(find(:element, "data-testid": "distribution-confirmation-partner")).to have_text("Test Partner")
```

And so on for all occurrences?

btw, the storage location factory already has a hard-coded name:
```ruby
name { "Smithsonian Conservation Center" }
```

Although using that in the distribution system spec expectations might create some confusion as to where that value from.

The partner factory on the other hand uses some variability, although with a sequence rather than faker:
```ruby
sequence(:name) { |n| "Leslie Sue, the #{n}" }
```

Reply:
> Did you mean to change the FactoryBot create to pass in specific names and then replace throughout the entire spec file?

Yep, exactly! The factory values are IMO something that should only be created but never referenced, because as you say it's really confusing as to where the values come from. If we need to check the value, we should always be providing it ourselves.

Reply:
Updated to specific partner and storage names.